src/webui/webapplication.cpp

```diff
@@ -774,6 +777,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
```

Comment:
>When a codebase is followed, it generally obviates code reviews like these. 

I don't think this is the case here, `splitRef()` is only on Qt > 5.4 which is available just recently (Qt 5.5 minimum now).
Following this logic, we should still code with Qt4.

>However nitpicking every line of a PR makes me a lot less eager to do so.

Sorry to have you feel this way, but then, how do I express my opinion? this is code review after all... ?